### PR TITLE
🌐 Add French translation for `docs/fr/docs/tutorial/index.md`

### PR DESCRIPTION
**Pull request by JulianMaurin**
Dear fellow translators 👋

This is a small PR to translate `docs/tutorial/index.md` to french 🇫🇷

Thanks 🍰

### CODE REVIEW

File: docs/fr/docs/tutorial/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    ```
+    pip install uvicorn
+    ```
+
+    Et la même chose pour chacune des dépendances facultatives que vous voulez utiliser.
+
+## Guide utilisateur avancé
+
+Il existe également un **Guide de utilisateur avancé** que vous pouvez lire plus tard après ce **Tutoriel - Guide de utilisateur**.
```
Comment (rjNemo) [inline docs/fr/docs/tutorial/index.md:74]:
```suggestion
Il existe également un **Guide d'utilisation avancé** que vous pouvez lire plus tard après ce **Tutoriel - Guide d'utilisation**.
```

File: docs/fr/docs/tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Le **Guide de l'utilisateur avancé**, qui s'appuie sur cette base, utilise les mêmes concepts et vous apprend quelques fonctionnalités supplémentaires.
```
Comment (rjNemo) [inline docs/fr/docs/tutorial/index.md:76]:
```suggestion
Le **Guide d'utilisation avancé**, qui s'appuie sur cette base, utilise les mêmes concepts et vous apprend quelques fonctionnalités supplémentaires.
```

File: docs/fr/docs/tutorial/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Mais vous devez d'abord lire le **Tutoriel - Guide de utilisateur** (ce que vous êtes en train de lire en ce moment).
```
Comment (rjNemo) [inline docs/fr/docs/tutorial/index.md:78]:
```suggestion
Mais vous devez d'abord lire le **Tutoriel - Guide d'utilisation** (ce que vous êtes en train de lire en ce moment).
```

File: docs/fr/docs/tutorial/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Il est conçu pour que vous puissiez construire une application complète avec seulement le **Tutoriel - Guide de utilisateur**, puis l'étendre de différentes manières, en fonction de vos besoins, en utilisant certaines des idées supplémentaires du **Guide de utilisateur avancé**.
```
Comment (rjNemo) [inline docs/fr/docs/tutorial/index.md:80]:
```suggestion
Il est conçu pour que vous puissiez construire une application complète avec seulement le **Tutoriel - Guide d'utilisation**, puis l'étendre de différentes manières, en fonction de vos besoins, en utilisant certaines des idées supplémentaires du **Guide d'utilisation avancé**.
```

**Review by rjNemo — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 0b42c565deb6f8943396a588e26e7074d22fe731 at: https://649a1347d1bbbc0f1c60fa9a--fastapi.netlify.app

**Review by axel584 — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @JulianMaurin! 🍰 

And thanks for the reviews as always @axel584 and @rjNemo ! ☕

**Pull request merged**